.npmignore

```diff
@@ -1,8 +1,12 @@
+.idea
```

Comment:
I know this was preexisting, but with al the stuff we're adding here - what do you think of using `package.json`'s `files` as a whitelist instead of this blacklist?